com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -3,6 +3,7 @@
 using System.Collections.Generic;
 using Unity.Collections;
 using System;
+using System.Linq;
```

Comment:
Is this leftover, I might be wrong, but I don't see any LINQ usage here?

Reply:
Line 468, `uniquePositions.Keys.ToArray();` requires linq

Reply:
👍